craft_application/fetch.py

```diff
@@ -104,6 +104,8 @@ def env(self) -> dict[str, str]:
             "REQUESTS_CA_BUNDLE": str(_FETCH_CERT_INSTANCE_PATH),
             # Same, but for cargo.
             "CARGO_HTTP_CAINFO": str(_FETCH_CERT_INSTANCE_PATH),
+            # Have go download directly from repositories
+            "GOPROXY": "direct",
```

Comment:
Is the problem that this solves that the fetch service would see all the requests going through, say, `proxy.golang.org` and not their original sources?

Reply:
@cmatsuoka can you clarify this go inspector restriction?

Reply:
We can take this offline :-)